packages/vehicle-manufacture-network/models/org.base.cto

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
not sure if "org.base.cto" is a good name for the model file now there is only one